devkit/samples/hooks/hooks.feature.ts

```diff
@@ -13,12 +13,12 @@ When('a step passes', function () {
   // no-op
 })
 
-When('a step throws an exception', function () {
+When('a step fails', function () {
   throw new Error('Exception in step')
 })
 
 After(function () {
-  throw new Error('Exception in hook')
+  // no-op
```

Comment:
This changes the behaviour - is there any loss of coverage?

Reply:
Because we want to actually test the steps being fired I figured it would be better to test the positive use cases by default and test the hook failures in the hook failures.

```
  Scenario: With a tag, a failure in the hook and a passed step
    When a step passes
```

Will generate the failure in the hook after the fact, so we don't lose any coverage, just the frequency of occurences.